data/performance.json

```diff
@@ -193,10 +194,13 @@
     "Office for Standards in Education - OFSTED": 1,
     "Office of the Public Guardian": 3,
     "Oil and Gas Authority": 1,
+    "Olton Mere Sailing Club (CASC)": 1,
     "Orkney Islands Council": 1,
     "Oxford Health NHS Foundation Trust": 4,
     "PCC For Cambridgeshire Police": 1,
     "PCC For Merseyside": 1,
+    "PCC for Surrey": 1,
+    "PCC for Sussex": 1,
```

Comment:
We have an organisation named `Police and Crime Commissioner for Sussex`, we should probably find out which one they prefer and combine them. cc @richblake

Reply:
👍